crates/net/network/src/session/mod.rs

```diff
@@ -473,6 +477,11 @@ impl SessionManager {
                 self.active_sessions.insert(peer_id, handle);
                 self.counter.inc_active(&direction);
 
+                match direction {
```

Comment:
lets also add `is_outgoing` here

https://github.com/paradigmxyz/reth/blob/6799fc3600601d4aadd73ea808734d8bd7f11cda/crates/net/network/src/session/mod.rs#L694-L697

and use this

Reply:
cool done